util/util.js

```diff
@@ -17,6 +17,7 @@
 const _ = require('lodash')
 
 const validDomainRegex = /^[a-zA-Z0-9-_.]+$/
+const hashDomainRegex = /^([A-Za-z0-9+/.]{4})*([A-Za-z0-9+/.]{3}=)?$/;
```

Comment:
zconfig.alibabausercontent.com.w.cdngslb.com
This domain will also match hash domain regular expression

Reply:
mapdownload.autonavi.com.danuoyi.tbcache.com